bootstrap/kubeadm/main.go

```diff
@@ -123,6 +125,12 @@ func InitFlags(fs *pflag.FlagSet) {
 	fs.DurationVar(&syncPeriod, "sync-period", 10*time.Minute,
 		"The minimum interval at which watched resources are reconciled (e.g. 15m)")
 
+	fs.Float32Var(&restConfigQPS, "kubernetes-client-qps", 20,
```

Comment:
Took a look at the kube-controller-manager, kube-scheduler and kube-proxy.

Looks like they are calling these flags `--kube-api-qps`and `--kube-api-burst`. Maybe good to align?

Reply:
Done :slightly_smiling_face: